documentation/content/en/book/02-concepts/_index.md

```diff
@@ -10,9 +10,8 @@ menu:
 ---
 
 ## What is kpt?
-###
 
-> kpt supports management of
+kpt supports management of
 [Configuration as Data](https://github.com/kptdev/kpt/blob/main/docs/design-docs/06-config-as-data.md).
```

Comment:
Shouldn't we merge this to the proper documentation? (Not in this pr, but if we agree I can create an issue for it)